docs/content/ingestion/index.md

```diff
@@ -150,6 +159,39 @@ This Avro bytes decoder first extract `subject` and `id` from input message byte
 | type | String | This should say `avro_1124_rest_client`. | no |
 | url | String | Specifies the endpoint url of your Avro-1124 schema repository. | yes |
 
+#### Avro Schema Mappings
+
+If `schemaMappings` is not included, no mappings are assumed and any field, either dimension or 
+metric, referenced in the parseSpec is assumed to be a top-level field in the incoming avro 
+record. On the other hand, if the avro record being ingested is complex in nature, containing arrays,
+maps, and sub-records, `schemaMappings` can be used to specify how to extract data from the complex
+structure. If 'schemaMappings` is present, it is considered to be a map, with keys being fields that
+might be of interest in the parseSpec and the values being the path through the avro record.
+
+The syntax for mappings matches the standard defined in Apache Commons-BeanUtils and follows the
+following grammer:
+
+```
+MAPPING : fieldName ( SEGMENT )*
+SEGMENT : '.' fieldName
```

Comment:
can you call this something other than SEGMENT, it might confuse druid users as SEGMENT has a very distinct meaning in druid world.